### ISSUE TITLE
[T3] Seeds especificas

### ISSUE THREAD

**Issue by jpintoa20** [IIC2513/Syllabus-S1-2024-1#197]
Hola
En el enunciado con respecto a las seeds dice:
"Adem ́as, es necesario generar seeds para poder probar la funcionalidad y la integridad de su aplicaci  ́on. Para esto,
deben crear seeds que incluyan, como m ́ınimo, tres instancias de la entidad User y dos registros de diario (Entry)
asociadas a cada usuario"

Eso de tres instancias es como sin usar faker como lo hicimos en la ayudantia cierto? Es más como en el seeder que hicimos, tambien en la ayudantia, para first movies:
await queryInterface.bulkInsert('Movies', [{
      title: 'Arrival',
      genre: 'Sci-fi/Thriller',
      description: 'Louise Banks, a linguistics expert, along with her team, must interpret the language of aliens who have come to Earth in a mysterious spaceship.',

Solo quiero confirmar, porque lo habia hecho con faker y me entro la duda

Gracias

**Comment by renatasantander** [comment]
Hola! Deben crear seeds manualmente como se vio en la ayudantía y luego correr las seeds, para poblar la base de datos.

**Comment by jpintoa20** [comment]
yaa, o sea en el seed de Entries debe ir ya definido el usuario en belongs_to y este debe estar ya definido en la seed de USers?

**Comment by renatasantander** [comment]
Claro, debes crear dos entries asociadas a cada usuario ya existente (6 en total).
